examples/changed-signal/App.lua

```diff
@@ -0,0 +1,43 @@
+--[[
+	Displays a TextBox and a TextLabel that reverses the TextBox's input text
+]]
+local Roact = require(game.ReplicatedStorage.Roact)
+
+local InputTextBox = require(script.Parent.InputTextBox)
+local ReversedText = require(script.Parent.ReversedText)
```

Comment:
Do you think we need this abstraction? It seems like for an example like this, we can probably just inline those objects here, since their transform is so simple. It might make the example a little bit more obvious!

Reply:
I was on the fence about this, but I guess we can use bigger examples as places to demonstrate how you might split things into files.

In that regard, should I converge all the examples? The stress test is also simple, but a bit beefier than this one.

Reply:
Actually I take it back it's not that bad either.